### PR TITLE
[ENG-3970] Add file download count to file list view

### PR DESCRIPTION
**Pull request by adlius**
-   Ticket: []
-   Feature flag: n/a

## Purpose

Screenshot from 2022-08-31 14-52-57

### CODE REVIEW

File: translations/en-us.yml [hunk 1]
```diff
@@ -1943,6 +1943,7 @@ osf-components:
         number_selected: '{numberOfFilesSelected} item(s) selected'
         clear_selection: 'Clear selection'
         checked_out_file: 'This file is checked out. Functionality is limited until it is checked in.'
+        download_count: '{count, plural, =0 {# Download} one {# Download} other {# Downloads}}'
```
Comment (brianjgeiger) [inline translations/en-us.yml:1946]: I think it should be `0 downloads` rather than `0 download`.

**Review by brianjgeiger — CHANGES_REQUESTED** [review]
One small change.

File: translations/en-us.yml (hunk 1 above)
Comment (adlius) [reply translations/en-us.yml:1946]: English is a joke. :rofl:

Comment (brianjgeiger) [reply translations/en-us.yml:1946]: I know, right?

**Review by brianjgeiger — APPROVED** [review]

**Review by futa-ikeda — APPROVED** [review]

**Pull request merged**